src/pytest_mock_resources/container/redis.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from pytest_mock_resources.compat import redis
+from pytest_mock_resources.container import (
+    ContainerCheckFailed,
+    get_container_fn,
+    get_docker_host,
+)
+
+# XXX: To become overwritable via pytest config.
```

Comment:
I'd like to avoid adding new fixtures which have global config like this anymore. Ideally this would more closely follow the postgres mechanism where the fixture exposes the connect credentials.

Reply:
Would you please expand on this, since the Postgres fixture uses a very similar `config` dict.

Reply:
Nowadays, the `pmr_credentials` attribute replaces the original need for it to be global, as it is.

so, i think you can just make this
```
@pytest.fixture
def pmr_redis_config():
```
or something like that, then parametrize it into the container fixture. This was only a module-level/global variable because we were originally directly importing it elsewhere for postgres.